### DISCUSSION TITLE
very thanks you!

### DISCUSSION THREAD

**Discussion by SiNaPsEr0x** [bpatrik/pigallery2#811] [Show and tell]
I've been looking for a solution for my qnap ts-231 based on armv7 for years and every application has dropped support... but surprisingly I found you!! and I can tell you that it is better than any other app I had already tried!! thanks for supporting armv7!!! I would really like to donate something but I can't find a way to do it...

**Comment by bpatrik** [comment]
Thank you very much! I'm happy that you like the app. Unfortunately, there
is no way to financially support the project, but nice feedback like this I
appreciate a lot :)
